[IMP] employees: making targeted edits for v15 and updating images (#5038)
odoo/documentation: content/applications/hr/employees/new_employee.rst

     :guilabel:`Levels` field enter `Beginner`, `Intermediate`, and `Expert`, with the
     :guilabel:`Progress` listed as `25`, `50`, and `100`, respectively. Then, either click
     :guilabel:`Save & Close` or :guilabel:`Save & New`.

       .. image:: new_employee/math-skills.png
          :align: center
          :alt: Add new math skills and levels with the skill types form.

Work Information tab
--------------------

The Work Information tab is where the employee's specific job related information is housed. Their
working schedule, various roles, who approves their specific requests (time off, timesheets, and
expenses), and specific work location details are listed here. Enter the following information for
the new employee.

- :guilabel:`Location`: Select the :guilabel:`Work Address` and :guilabel:`Work Location` from the
  corresponding drop-down menus. The work address :guilabel:`External Link` button opens up the
  selected company form in a window, and allows for editing. The :guilabel:`Work Location` is the
  specific location details, such as a floor or building. If a new work location is needed, add the
  location by typing it in the field.

> - :guilabel:`Approvers`: Using the drop-down menus, select the employees responsible for approving

Reviewer: ```suggestion
- :guilabel:`Approvers`: Using the drop-down menus, select the users responsible for approving
```
small change, but since you can only choose a user here, not just an employee, I thought it made sense to make that distinction.

  to scroll to the desired month, then click on the day to select it.
- :guilabel:`Description`: Enter any relevant details in the field.

Once all the information is entered, click the :guilabel:`Save & Close` button if there is only one
entry to add, or click the :guilabel:`Save & New` button to save the current entry and create
another resumé line.

.. note::
   After the new employee form is saved, the current position and company is automatically added to

>    the :guilabel:`Resumé` tab, with the end date listed as *current*.
> 

Reviewer 2: Just want to expand on the context here as well as update to guilabels :)
```suggestion
   the :guilabel:`Resumé` tab as :guilabel:`Experience`, with the end date listed as
   :guilabel:`Current`.

```

  appear in a drop-down menu. For example, selecting :guilabel:`Language` as the :guilabel:`Skill
  Type` presents a variety of languages to select from under the :guilabel:`Skills` field. Select
  the appropriate pre-configured skill, or type in a new one.
- :guilabel:`Skill Level`: Pre-defined skill levels associated with the selected :guilabel:`Skill
  Type` appear in a drop-down menu. Select a skill level, then the progress bar automatically
  displays the pre-defined progress for that skill level. Skill levels and progress can be modified
  in the :guilabel:`Skill Level` pop-up form, which is accessed via the :guilabel:`External Link`
  button next to :guilabel:`Skill Level` field.

Once all the information is entered, click the :guilabel:`Save & Close` button if there is only one
entry to add, or click the :guilabel:`Save & New` button to save the current entry and create
another skill.

To delete any line from the :guilabel:`Resumé` tab, click the :guilabel:`🗑️ (trash can)` icon to
delete the entry. Add a new line by clicking the :guilabel:`Add` button next to the corresponding
section.

.. _employees/skill-types:

> Skill Types

Reviewer 2: Sentence case headings unless a proper noun :)
```suggestion
Skill types
```

form appears. Fill out all the details and then click :guilabel:`Save`. Repeat this for all the
skill types needed.

- :guilabel:`Skill Type`: Enter the name of the skill type. This should be somewhat generic, since
  the specific skills listed will be housed under this category.
- :guilabel:`Skills`: Click :guilabel:`Add a line` and enter the information for the new skill, then
  repeat for all other needed skills.
- :guilabel:`Levels`:  Click :guilabel:`Add a line` and a :guilabel:`Create Levels` form appears.
  Enter the name of the level, and assign a percentage (0-100) for that level. Click :guilabel:`Save
  & New` to save the entry and add another level, or click :guilabel:`Save & Close` to save the
  level and close the form.

  .. example::
     To add a math skill set, enter `Math` in the :guilabel:`Name` field. Next, in the
     :guilabel:`Skills` field, enter `Algebra`, `Calculus`, and `Trigonometry`. Last, in the
     :guilabel:`Levels` field enter `Beginner`, `Intermediate`, and `Expert`, with the
     :guilabel:`Progress` listed as `25`, `50`, and `100`, respectively. Then, either click
     :guilabel:`Save & Close` or :guilabel:`Save & New`.

>        .. image:: new_employee/math-skills.png
>           :align: center
>           :alt: Add new math skills and levels with the skill types form.

Reviewer 2: Image alignment, need to scoot back a few spaces to be inline with the `e` in `.. example::`
```suggestion
     .. image:: new_employee/math-skills.png
        :align: center
        :alt: Add new math skills and levels with the skill types form.
```

> Work Information tab

Reviewer 2: Sentence case heading
```suggestion
Work information tab
```

--------------------

> The Work Information tab is where the employee's specific job related information is housed. Their
> working schedule, various roles, who approves their specific requests (time off, timesheets, and
> expenses), and specific work location details are listed here. Enter the following information for
> the new employee.

Reviewer 2: Need a guilabel on the UI text during the active instruction
```suggestion
The :guilabel:`Work Information` tab is where the employee's specific job related information is
housed. Their working schedule, various roles, who approves their specific requests (time off,
timesheets, and expenses), and specific work location details are listed here. Enter the following
information for the new employee.
```

  required) for the employee. The :guilabel:`External Link` button opens up a detailed view of the
  specific daily working hours. Working hours can be modified or deleted here. Click
  :guilabel:`Save` to save any changes.
- :guilabel:`Planning`: The :guilabel:`Planning` section affects the *Planning* app, and will only
  appear if the *Planning* app is installed. Click on a planning role from the drop-down menu for
  both the :guilabel:`Default Planning Role` and the :guilabel:`Planning Roles` fields to add a
  role. There is no limit to the amount of :guilabel:`Planning Roles` that can be selected for an
  employee, but there can only be one :guilabel:`Default Planning Role`. The default is the
  *typical* role that the employee performs, where the :guilabel:`Planning Roles` are *all* the
  specific roles the employee is able to perform.

  .. image:: new_employee/work-info.png
     :align: center
     :alt: Add the work information to the Work Information tab.

.. tip::
   The users that appear in the drop-down menu for the :guilabel:`Approvers` section must have
   *Administrator* rights set for the corresponding human resources role. To check who has these

>    rights, go to :menuselection:`Settings app --> Users --> → Manage Users`. Click on an employee,

Reviewer 2: Not sure if it was intentional to leave the → in Manage Users, but it formats weird and I think it is more readable without it 
![Screenshot 2023-10-19 at 5 40 31 PM](https://github.com/odoo/documentation/assets/36018073/7942a535-fd96-4853-ba74-4d4e09f0dae0)

```suggestion
   rights, go to :menuselection:`Settings app --> Users --> Manage Users`. Click on an employee,
```

   and check the :guilabel:`Human Resources` section of the :guilabel:`Access Rights` tab.

>    - In order for the user to appear as an approver for :guilabel:`Expenses`, they must have either
>      :guilabel:`Team Approver`, :guilabel:`All Approver`, or :guilabel:`Administrator` set for the
>      :guilabel:`Expenses` role.
> 
>    - In order for the user to appear as an approver for :guilabel:`Time Off`, they must have either
>      :guilabel:`Officer` or :guilabel:`Administrator` set for the :guilabel:`Time Off` role.
> 
>    - In order for the user to appear as an approver for :guilabel:`Timesheets`, they must have
>      either :guilabel:`Manager`, :guilabel:`Officer`, or :guilabel:`Administrator` set for the
>      :guilabel:`Payroll` role.

Reviewer 2: No need for new lines between bullet points of the same level
```suggestion
   - In order for the user to appear as an approver for :guilabel:`Expenses`, they must have either
     :guilabel:`Team Approver`, :guilabel:`All Approver`, or :guilabel:`Administrator` set for the
     :guilabel:`Expenses` role.
   - In order for the user to appear as an approver for :guilabel:`Time Off`, they must have either
     :guilabel:`Officer` or :guilabel:`Administrator` set for the :guilabel:`Time Off` role.
   - In order for the user to appear as an approver for :guilabel:`Timesheets`, they must have
     either :guilabel:`Manager`, :guilabel:`Officer`, or :guilabel:`Administrator` set for the
     :guilabel:`Payroll` role.
```

.. note::
   :guilabel:`Working Hours` are related to a company's working times, and an employee cannot have
   working hours that are outside of a company's working times.

   Each individual working time is company-specific, so for multi-company databases, each company
   needs to have its own working hours set.

   If an employee's working hours are not configured as a working time for the company, new working
   times can be added, or existing working times can be modified. To add or modify a working time,
   go to the :menuselection:`Payroll app --> Configuration --> Working Times`, and add a new working
   time or edit an existing one.

   After the new working time is created, set the working hours for the employee.

> Private Information tab

Reviewer 2: Sentence case headings
```suggestion
Private information tab
```

     :align: center
     :alt: Add the work information to the Work Information tab.

.. tip::
   The employees that appear in the drop-down menu for the :guilabel:`Approvers` section must have
   *Administrator* rights set for the corresponding human resources role. To check who has these
   rights, go to :menuselection:`Settings app --> Users --> → Manage Users`. Click on an employee,
   and check the :guilabel:`Human Resources` section of the :guilabel:`Access Rights` tab.

>    - In order for the employee to appear as an approver for :guilabel:`Expenses`, they must have

Reviewer: ```suggestion
   - In order for the user to appear as an approver for :guilabel:`Expenses`, they must have
```

     either :guilabel:`Team Approver`, :guilabel:`All Approver`, or :guilabel:`Administrator` set for
     the :guilabel:`Expenses` role.

>    - In order for the employee to appear as an approver for :guilabel:`Time Off`, they must have

Reviewer: ```suggestion
   - In order for the user to appear as an approver for :guilabel:`Time Off`, they must have
```

     either :guilabel:`Officer` or :guilabel:`Administrator` set for the :guilabel:`Time Off` role.

>    - In order for the employee to appear as an approver for :guilabel:`Timesheets`, they must have

Reviewer: ```suggestion
   - In order for the user to appear as an approver for :guilabel:`Timesheets`, they must have
```

- :guilabel:`Private Contact`: Enter the personal :guilabel:`Address` for the employee. The
  selection can be made with the drop-down menu. If the information is not available, type in the
  name for the new address. To edit the new address, click the :guilabel:`External Link` button to
  open the address form. On the address form, enter the necessary details, then click
  :guilabel:`Save`

  Some other information in the :guilabel:`Private Contact` section may auto-populate, if the
  address is already listed in the drop-down menu.

  Next, enter the employee's :guilabel:`Email` address and :guilabel:`Phone` number in the
  corresponding fields.

  Select the employee's preferred :guilabel:`Language` from the drop-down menu.

  Enter the employee's :guilabel:`Bank Account Number` using the drop-down menu. If the bank is not
  already configured (the typical situation when creating a new employee) enter the bank account
  number, and click :guilabel:`Create and Edit`. A :guilabel:`Create: Bank Account Number` for
  appears. Fill in the information, then click :guilabel:`Save`.

  Finally, enter the :guilabel:`Home-Work Distance` in the field. This field is only necessary if

>   the employee is receiving any type of commuter benefits.

Reviewer 2: Add a blank line before starting bullet points
```suggestion
  the employee is receiving any type of commuter benefits.
  
```

- :guilabel:`Marital Status`: Select either :guilabel:`Single`, :guilabel:`Married`,
  :guilabel:`Legal Cohabitant`, :guilabel:`Widower`, or :guilabel:`Divorced` from the drop-down
  menu.

> - :guilabel:`Emergency`: Type in the name and phone number of the employee's emergency
>   contact.

Reviewer 2: Early line break :)
```suggestion
- :guilabel:`Emergency`: Type in the name and phone number of the employee's emergency contact.
```

  the file explorer, and click :guilabel:`Open`.

.. image:: new_employee/private-info.png
   :align: center
   :alt: Add the private information to the Private Information tab.

HR Settings tab
---------------

This tab provides various fields for different information, depending on the country the company is
located. Different fields are configured for different locations, however some sections appear
regardless.

- :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a  :ref:`Related User
  <employees/related-user>`, with the drop-down menus.
- :guilabel:`Payroll`: Select the :guilabel:`Current Contract` and :guilabel:`Job Position` from the
  drop-down menus. If applicable, enter the :guilabel:`Registration Number` in this section.
- :guilabel:`Previous Employer`: This section appears only for Belgian companies, and will not be
  visible for other locations. These are days that will be paid to the new employee. Enter any
  :guilabel:`Simple Holiday Pay to Recover`, :guilabel:`Number of Days to recover`, and
  :guilabel:`Recovered Simple Holiday Pay` from a previous employer, for both N and N-1 categories.

> - :guilabel:`Previous Occupations`: This section appears ony for Belgian companies, and will not be

Author: I have this at the top : This tab provides various fields for different information, depending on the country the company is
located. Different fields are configured for different locations, however some sections appear
regardless.
I think that might suffice since it was in the 14 and 16 docs, too.

  access the year ranges. Use the :guilabel:`< (left)` and :guilabel:`> (right)` arrow icons,
  navigate to the correct year range, and click on the year. Next, click on the month. Last, click
  on the day to select the date. Type in the information for the :guilabel:`Identification No`,
  :guilabel:`Passport No`, and :guilabel:`Place of Birth` fields.
- :guilabel:`Dependant`: If the employee has any children, enter the :guilabel:`Number of Children`
  in the field.
- :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
  Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.
  Using the calendar module, select the :guilabel:`Visa Expire Date` and/or the :guilabel:`Work
  Permit Expiration Date` to enter the expiration date(s). If available, upload a digital copy of
  the work permit document. Click :guilabel:`Upload Your File`, navigate to the work permit file in
  the file explorer, and click :guilabel:`Open`.

.. image:: new_employee/private-info.png
   :align: center
   :alt: Add the private information to the Private Information tab.

> HR Settings tab

Reviewer 2: Sentence case headings
```suggestion
HR settings tab
```

---------------

This tab provides various fields for different information, depending on the country the company is
located. Different fields are configured for different locations, however some sections appear
regardless.

> - :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a  :ref:`Related User

Reviewer 2: Remove extra space
```suggestion
- :guilabel:`Status`: Select an :guilabel:`Employee Type` and, if applicable, a :ref:`Related User
```

  <employees/related-user>`, with the drop-down menus.
- :guilabel:`Payroll`: Select the :guilabel:`Current Contract` and :guilabel:`Job Position` from the
  drop-down menus. If applicable, enter the :guilabel:`Registration Number` in this section.
- :guilabel:`Previous Employer`: This section appears only for Belgian companies, and will not be
  visible for other locations. These are days that will be paid to the new employee. Enter any
  :guilabel:`Simple Holiday Pay to Recover`, :guilabel:`Number of Days to recover`, and
  :guilabel:`Recovered Simple Holiday Pay` from a previous employer, for both N and N-1 categories.
- :guilabel:`Previous Occupations`: This section appears ony for Belgian companies, and will not be
  visible for other locations. Click :guilabel:`Add a line` to enter information for each previous
  occupation. Enter the number of :guilabel:`Months`, the :guilabel:`Amount`, and the

>   :guilabel:`Occupational Rate` in the corresponding fields. Click the  :guilabel:`🗑️ (trash can)`

Reviewer 2: Remove extra space
```suggestion
  :guilabel:`Occupational Rate` in the corresponding fields. Click the :guilabel:`🗑️ (trash can)`
```

  Enter the employee's cost per hour in a $XX.XX format. This is factored in when the employee is
  working at a :doc:`work center
  <../../inventory_and_mrp/manufacturing/management/using_work_centers>`. This value affects the
  manufacturing costs for a product, if the value of the manufactured product is not a fixed amount.

.. image:: new_employee/hr-settings.png
   :align: center
   :alt: Enter any information prompted in the HR Settings tab for the employee.

.. _employees/related-user:

.. important::
   Employees do not also need to be users. An employee does **not** count towards billing, while
   *Users* **do** count towards billing. If the new employee should also be a user, the user must
   be created.

   In the :guilabel:`Related User` field, type in the name of the user to add, then click
   :guilabel:`Create and Edit...`. A :guilabel:`Create: Related User` form appears. Type in the
   :guilabel:`Name`, :guilabel:`Email Address`, and then select the :guilabel:`Company` from the
   drop-down menu. Click :guilabel:`Save` after the information is entered. Once the record is
   saved, the new user appears in the :guilabel:`Related User` field.

Documents
=========

> All documents associated with an employee are stored in the :guilabel:`Documents` app. The number of
> documents associated with the employee appear in the :guilabel:`Documents` smart button on the
> employee form. Click on the smart button, and all the documents appear. For more information on the
> :guilabel:`Documents` app, refer to :doc:`this document </applications/finance/documents>`.

Reviewer 2: Update guilables to italics and use a context-ful link description please :)
```suggestion
All documents associated with an employee are stored in the *Documents* app. The number of documents
associated with the employee appear in the :guilabel:`Documents` smart button on the employee form.
Click on the smart button, and all the documents appear. For more information on the *Documents*
app, refer to the :doc:`Documents documentation </applications/finance/documents>`.
```
